.gitignore

```diff
@@ -15,6 +15,7 @@ coverage_report/
 .coverage
 /env/
 *.egg-info/
+*.DS_Store
```

Comment:
Remove this in favor of a global gitignore, which you can set up following this: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html
The idea being that gitignore entries in the repo should all be python and code specific, not anything related to your OS or an IDE, etc.

Reply:
gotcha, made the change